### DISCUSSION TITLE
SEQ - Crashes - The process was terminated due to an unhandled exception - exception code c0000005

### DISCUSSION THREAD

**Discussion by jorgenfries** [datalust/seq-tickets#2092] [Help]
We have several SEQ instances running, and updated last week to the latest version [_2024.1.11001_]
On two different instances with heavy load we have seen memory access violation crashes.

On server 1 we can reproduce the error by choosing a specific signal or filtering for a specific application name in the log stream.

On server 2 we see the crash intermittently(2 times for the last 24 hours).

SEQ is running on Windows 2019 servers

Output from Eventlog:
```
seq.exe 
   2024.1.11001.0 
   65680000 
   flarec.dll 
   0.0.0.0 
   65c17e29 
   c0000005 
   00000000002fe392 
   3104 
   01da5ce80e1d0e47 
   C:\Program Files\Seq\seq.exe 
   C:\Program Files\Seq\Native\flarec.dll 
   af45aed0-5854-4209-8b85-8d168075aa5e 


Faulting application name: seq.exe, version: 2024.1.11001.0, time stamp: 0x65680000
Faulting module name: flarec.dll, version: 0.0.0.0, time stamp: 0x65c17e29
Exception code: 0xc0000005
Fault offset: 0x00000000002fe392
Faulting process id: 0x3104
Faulting application start time: 0x01da5ce80e1d0e47
Faulting application path: C:\Program Files\Seq\seq.exe
Faulting module path: C:\Program Files\Seq\Native\flarec.dll
Report Id: af45aed0-5854-4209-8b85-8d168075aa5e
Faulting package full name: 
Faulting package-relative application ID: 

Application: seq.exe
CoreCLR Version: 8.0.123.58001
.NET Version: 8.0.1
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 00007FFC43C0E392
```

We can't see any serverlogs for SEQ itself - In this example we are using E:\ as the SEQ datastore.
E:\SeqData\Logs 

Does not have any new logs, last logs present here are from last year when we migrated and updated SEQ from previous versions.

**Comment by jorgenfries** [comment]
Update - We can trigger the crash just by going to Dashboard --> Overview

**Reply by nblumhardt** [reply]
Thanks for the report, @jorgenfries - as you noted, this is a duplicate of https://github.com/datalust/seq-tickets/issues/2086, we'll follow up over there 👍

**Reply by nblumhardt** [reply]
(Just a quick note, even with storage set to a custom location, logs will be written to `C:\ProgramData\Seq\Logs` on Windows. HTH!)

**Reply by nblumhardt** [reply]
Hi @jorgenfries - sorry, I've just checked over the details in your post and in fact this one is different from #2036. I'll create a new ticket for it.

Would it be possible to contact `support@datalust.co` so we can coordinate there? We'll need to walk through creating a crash dump with you so that we can pinpoint this issue.

Best regards,
Nick

**Reply by nblumhardt** [reply]
Ah, and now I see you've already emailed me; traveling this morning so all a little disjointed, sorry :-)

**Reply by nblumhardt** [reply]
Just letting you know that we're still actively investigating this.
